backend/src/domain/models/Game.js

```diff
@@ -55,20 +55,13 @@ export class Game {
    * of the session
    */
   async nextRound() {
-    this.round++;
-    if (this.round === this.swipeDeck.length) {
-      this.endGame();
-      return;
-    }
-
-
-    for (let i=1; i < this.swipeDeck.length; i++) {
```

Comment:
If there was only 1 restaurant, would have skipped the for loop and immediately ended the game